ui/pages/settings/networks-tab/networks-form/networks-form.js

```diff
@@ -213,6 +243,27 @@ const NetworksForm = ({
     dispatch,
   ]);
 
+  const autoSuggestTicker = useCallback((formChainId) => {
+    const decimalChainId = getDisplayChainId(formChainId);
+    if (
+      !decimalChainId ||
```

Comment:
What condition does this cover?  `0`?

Reply:
Good catch, I think getDisplayChainId already accounts for this condition